docs/vertx-runtime/master.adoc

```diff
@@ -97,7 +97,7 @@ include::topics/assembly_cache-mission-booster.adoc[leveloffset=+2]
 
 :context: vertx
 
-== Application Development
+== Application development
```

Comment:
I suspect I might tbe the one to have introduced this int he first place, but, I think we should change the title here to be consistent with the gerund forms used for other headings:

How 'bout:

> Developing a {runtime} application.

I think Swarm already uses the gerund title here.

Reply:
Fixed in a new commit.